### PR TITLE
[SHOT-3859] Scene breakdown supports multiple entities

### PR DESCRIPTION
**Pull request by barbara-darkshot**

### CODE REVIEW

**Review by rob-aitchison — DISMISSED** [review]
Looks good!

File: hooks/tk-multi-breakdown/basic/scene_operations.py [hunk 1]
```diff
@@ -42,10 +42,15 @@ def scan_scene(self):
 
         refs = []
 
-        reference_template = self.parent.engine.get_template("reference_template")
+        # reference_template = self.parent.engine.get_template("reference_template")
```
Comment (staceyoue) [inline hooks/tk-multi-breakdown/basic/scene_operations.py:45]: Should this line be removed?

**Review by staceyoue — DISMISSED** [review]
Just one comment, otherwise LGTM!

File: hooks/tk-multi-breakdown/basic/scene_operations.py (hunk 1 above)
Comment (barbara-darkshot) [reply hooks/tk-multi-breakdown/basic/scene_operations.py]: Thanks Stacey, I didn't pay attention to this line :)

**Review by staceyoue — APPROVED** [review]
LGTM!

**Pull request merged**